packages/app/src/systems/Chains/fuel/hooks/useFuelAccountConnection.tsx

```diff
@@ -54,7 +54,10 @@ export const useFuelAccountConnection = () => {
   return {
     handlers: {
       connect: handleConnect,
-      disconnect,
+      disconnect: () => {
+        disconnect();
+        localStorage.clear();
```

Comment:
by clearing the whole `localstorage` you will clean connectkit info too and disconnect eth wallet if connected

this [task](https://linear.app/fuel-network/issue/FRO-272/using-caching-on-tx-history-fetch) aims to remove transaction data only. which can be done by cleaning only needed localStorage data (if we can discover).

idea would be to create a `localStorage.ts` file inside of `Chains/etc/utils`, to centralize this behaviors.
<img width="610" alt="image" src="https://github.com/FuelLabs/fuels-portal/assets/8636507/dea16bb3-fea6-458f-b22c-8a03be56b873">

so would have `get / setBlockDate` `get / setHashDone`, 

will also have a `clean` method, which will iterate using `localStorage.length` and `localStorage.key` to discover which localStorage items are ours, and delete them using `localStorage.removeItem`